README.md

```diff
@@ -122,7 +124,38 @@ Then run `test/run_tests.sh` and `test/cpp/run_tests.sh` to verify the setup is
 
 
 
-# How To Install Pre Built PyTorch TPU Wheels
+# Using Pre Built Releases 
+
+## Pre Built Docker Images (recommended)
+
+Docker images with `torch` and `torch_xla` preinstalled in the `pytorch` conda
+environment are distributed under: `gcr.io/tpu-pytorch/xla`. This image has two type of tags which take the forms of:
+
+* gcr.io/tpu-pytorch/xla:nightly
+* gcr.io/tpu-pytorch/xla:YYYYMMDD (ex. gcr.io/tpu-pytorch/xla:nightly_20190531)
+
+With these images, for example, you can train mnist on TPUs with the following commands:
+
+```Shell
+docker pull gcr.io/tpu-pytorch/xla:nightly
+```
+
+After pulling the image you can either:
+
+* Run the container with a command:
+```Shell
+docker run --shm-size 16G -e XRT_TPU_CONFIG="tpu_worker;0;<IP of the TPU node>:8470" gcr.io/tpu-pytorch/xla:nightly python pytorch/xla/test/test_train_mnist.py
+```
+
+* Run the script in an interactive shell:
+```Shell
+docker run -it --shm-size 16G gcr.io/tpu-pytorch/xla:nightly
```

Comment:
Is `/bin/sh` implicit?
I always passed it to `docker run -it ...`

Reply:
I believe so, but it depends on the docker image's predefined entrypoint and debian's default I think is sh and we build off of that.